kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -629,6 +629,12 @@ net8021xUsername=Identity (Username)
 net8021xUsernameHelp=A String that represents a username used for EAP authentication.
 net8021xPassword=Password
 net8021xPasswordHelp=A String that represents a password used for EAP authentication.
+net8021xKeystorePid=Keystore Pid
+net8021xKeystorePidHelp=The Pid of the Keystore that contains the Certificates and Private Keys used for EAP authentication.
+net8021xCaCert=Certificate Authority Certificate (CA-Cert) - Optional
+net8021xCaCertHelp=The name of the Certificate which will be used for EAP authentication.
+net8021xPrivateKey=Client Private Key
```

Comment:
This is not a Private Key... it's a "Public/Private Key **Pair**". Same goes for each and every variable name you used. I think it's misleading.